### DISCUSSION TITLE
Erfahrungsbericht Open3E mit iobroker - eine Art Doku

### DISCUSSION THREAD

**Discussion by philippoo66** [open3e/open3e#8] [Show and tell]
mit allerbestem Dank an Hotziplotzi hier sein Leitfaden zum Dranlanghangeln

![grafik](https://github.com/abnoname/open3e/assets/122479122/04c74ca1-9766-4987-bc9e-9f84ffd8ddce)

![grafik](https://github.com/abnoname/open3e/assets/122479122/b07d2349-6574-430d-b0da-b84dfa542aab)

![grafik](https://github.com/abnoname/open3e/assets/122479122/de9e175a-2742-4959-a713-97a8e3af7a3f)

![grafik](https://github.com/abnoname/open3e/assets/122479122/18a68697-146d-48bd-b7e2-d43dd564a7bd)

![grafik](https://github.com/abnoname/open3e/assets/122479122/e7b32299-fe99-460a-869d-80e5f5adc0d7)

![grafik](https://github.com/abnoname/open3e/assets/122479122/c0fd1c2e-bcb4-45c8-85b7-1f151e4a25e7)

**Comment by Dreman65** [comment]
Einige Hinweise meinerseits auf Basis der ersten Lese- und Schreibversuche:

- Bei mir läuft das problemlos auf einem Raspi 3B+ mit 32-bit Raspberry Pi OS Lite zusammen mit iobroker, history als Datenbank und flot/vis für die visuelle (zeitliche) Darstellung der Datenpunkte.
- Die Stecker hatte ich seinerzeit (relativ günstig) übers [Internet](https://www.ebay.de/itm/235143048291?_trkparms=amclksrc%3DITM%26aid%3D1110025%26algo%3DHOMESPLICE.COMPOSITELISTINGS%26ao%3D1%26asc%3D20220405142716%26meid%3D9e928d064399463ebffd0ce39565db7b%26pid%3D101506%26rk%3D1%26rkt%3D25%26sd%3D235093533265%26itm%3D235143048291%26pmt%3D0%26noa%3D1%26pg%3D3458402%26algv%3DAlgoIndex5SimRanker%26brand%3DMarkenlos&_trksid=p3458402.c101506.m1851) bezogen.
- Besonders interessant finde ich für Vitodens 300-W-B3HG Besitzer den Datenpunkt 919 OutsideTemperature DumpingFactor, mit dem sich (entgegen den offiziellen Viessmann-Aussagen) auch bei der aktuellen Vitodens-Version die Zeitkonstante für die Berechnung der CalculatedOutsideTemperature einstellen lässt (Werkseinstellung 128 = 21,3 Stunden). Damit lässt sich die Heizung auch an weniger gut gedämmte Gebäude optimal anpassen (wie es bei den vorigen Vitodens-Versionen) möglich war. Es steht zwar eventuell (nach meinen ersten Schreibversuchen) nicht der gesamte Einstellbereich (1 bis 199) zur Verfügung, aber es ist möglich als kleinsten Wert 18 (also 3 Stunden) zu schreiben, was für unsanierte Gebäude eine wesentliche Verbesserung gegenüber dem aktuellen Festwert 128 darstellt.
- Ebenso interessant der Datenpunkt 1791 DiverterValveDefaultPositionConfiguration (Ruhestellung des 3-Wege-Ventils, Werkseinstellung WW), der sich jetzt (ohne Viessmann Softwaretool) auch auf 'Heizen' stellen lässt und damit eine eventuelle Schwerkraftzirkulation speziell im 'Nur-WW-Modus' unterbindet.

**Reply by Hotziplotzi** [reply]
Gute Punkte!
Ich fand ja alleine schon den Zugriff auf die von der Heizung selbst berechnete gedämpfte Außentemperatur fein. Die Anzahl der 10-Minuten-Dämpfungsschritte wenigsten etwas einstellen zu können ist gut.
In der API (selbst bezahlt) gibts *nur* den gemessenen Datenpunkt "heating.sensors.temperature.outside".
Siehe: https://documentation.viessmann.com/static/iot/data-points
